apps/editing-toolkit/editing-toolkit-plugin/coming-soon/fallback-coming-soon-page.php

```diff
@@ -16,7 +16,8 @@
  * @return string Language slug
  */
 function get_current_locale() {
-	return function_exists( 'get_blog_lang_code' ) ? get_blog_lang_code() : get_locale();
+	$language = function_exists( 'get_blog_lang_code' ) ? get_blog_lang_code() : get_locale();
```

Comment:
why not move the `language` check into `get_iso_639_locale` since we use it in both places?